compiler/x/codegen/BinaryEvaluator.cpp

```diff
@@ -1937,13 +1937,59 @@ TR::Register *OMR::X86::TreeEvaluator::integerMulEvaluator(TR::Node *node, TR::C
    TR::Node     *firstChild     = node->getFirstChild();
    TR::Node     *secondChild    = node->getSecondChild();
    bool          nodeIs64Bit    = TR::TreeEvaluator::getNodeIs64Bit(node, cg);
+   bool isMemRef                = firstChild->getOpCode().isMemoryReference();
+   TR_X86OpCodes opCode, regOpCode, memOpCode, movOpCode;
```

Comment:
Leaving local uninitialized is not generally good form - could you initialize these to the badOpCode and then assert fatal after the switch to make sure that they have been set to a non-badOpCodeValue?